modules/oxide/src/less/theme/components/statusbar/statusbar.less

```diff
@@ -190,6 +190,10 @@
       }
     }
   }
+
+  .tox-statusbar__resize-handle:only-child {
+    flex: 1;
```

Comment:
This will make the resize handle be the width of the editor not just the corner of the editor. There are other way we could fix this one is to set justify-content: end on the parent container.
Or you could use `margin-left: auto` so that it has a margin of the full width in the container probably the simplest one.
Another is to keep a spacing element in there the `tox-statusbar__text-container` could be kept in there and be empty to push out the area so that the resize element is at the end.
Also remember to test with RTL it seem that the `margin-left: auto` works better than the `flex` approach here not sure why. Copy over the ar.js file into `js/tinymce/langs` then set `language: 'ar`` to test RTL.

Reply:
I went with `margin-left: auto`, tested with RTL ✅

Reply:
That suggests there wasn't a bug in RTL 🤔 

If there was, you'd have to use `margin-inline-start` instead

Reply:
It wasn't, the bug occurred for both LTR and RTL